client/Mac/CMakeLists.txt

```diff
@@ -108,6 +108,16 @@ endforeach()
 
 set_property(TARGET ${MODULE_NAME} PROPERTY FOLDER "Client/Mac")
 
+if(CMAKE_GENERATOR MATCHES "Ninja")
+	if (NOT ${CONFIGURATION} STREQUAL "")
+		set(safe_configuration "$CONFIGURATION")
+	else()
+		set(safe_configuration "")
+	endif()
+else()
+	set(safe_configuration "$(CONFIGURATION)")
```

Comment:
`$(CONFIGURATION)` and not `${CONFIGURATION}`?

Reply:
Unfortunately it's not that obvious with the configuration. How it was created $(CONFIGURATION) is used directly in the generator when a custom command is invoked. That's why it's $(CONFIGURATION) for make and $CONFIGURATION for ninja.